frontend/src/ClientDataForm.js

```diff
@@ -56,9 +62,9 @@ const ClientDataForm = () => {
           //alert('Form successfully submitted');
           var result = window.confirm("Form successfully submitted");
           if (result) {
```

Comment:
Do we need the `if` statement here?
Looks like whether `result` is True or not, we wanted to `window.location.href='/selectlanguage';` anyways so I think we shouldn't need an `if` statement here.